libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -34,13 +34,14 @@ void harness()
      * MQTT_GetIncomingPacketTypeAndLength(). */
     NetworkContext_t networkContext;
 
-    __CPROVER_assume( networkContext != NULL );
```

Comment:
NetworkContext_t is typedef'd above to an int and not to a pointer.

Reply:
This is deleted.